ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -216,7 +216,8 @@ export const ConceptHomepage = withCurrentWorkspace()(
     }
 
     componentDidMount() {
-      this.loadDomainsAndSurveys();
+      workspacesApi().updateRecentWorkspaces(this.props.workspace.namespace, this.props.workspace.id)
```

Comment:
Should updateRecentWorkspaces be called with any visit? Or just with edits? If its just with edits, I don't think we want this here, and if it is with any visit, could we just update on fetch to `getWorkspace`?

Reply:
With any visit. I would strongly prefer to update only on explicit UI actions so that some developer doesn't come along six months from now and not realize `getWorkspace` has a side effect.

Reply:
Talked offline. This is probably doable with a HOC or wrapper component, but we are spinning that off into a separate tech debt ticket.

Reply:
What's the rationale for filing a tech debt ticket instead of just putting this into workspace-wrapper right now? It seems pretty directly linked to what we're attempting to track here, and it would be a much smaller change.

Reply:
Based on the current UI stack, if there's a workspace page which is NOT part of workspace-wrapper, that's a bug. That's because workspace-wrapper is what controls setting `currentWorkspaceStore`, which is where all the React components get workspace data (e.g. `withCurrentWorkspace()`).

Worth auditing if it can be done quickly, but also reasonable for the purposes of this PR to assume that everything can or will be covered.

Reply:
Looks like the thing we thought was outside the workspace wrapper component just minimizes the header is the only thing the workspace wrapper component renders. I'll move the call there.